README.md

```diff
@@ -343,6 +343,18 @@ Default: `eni`
 
 Specifies the veth prefix used to generate the host-side veth device name for the CNI. The prefix can be at most 4 characters long.
 
+---
+
+`ADDITIONAL_ENI_TAGS`
+
+Type: String
+
+Default: `{}`
+
+Example values: `{"tag_key": "tag_val"}`
+
+Metadata applied to ENI help you categorize and organize your resources for billing or other purposes. Each tag consists of a custom-defined key and an optional value. Tag keys can have a maximum character length of 128 characters. Tag values can have a maximum length of 256 characters.
```

Comment:
Might consider adding one more sentence in here to make it clear that *all* ENIs on the host will be tagged with these tags.

Reply:
Sure, let me add that line in there.